src/main/com/fulcrologic/fulcro/raw/components.cljc

```diff
@@ -358,7 +358,10 @@
                        (-> class-or-factory meta :qualifier)
                        nil)
            queryid   (if (component-instance? class-or-factory)
-                       (get-query-id class-or-factory)
+                       ; hooked use-fulcro components are component instances, without comp/factory's :fulcro$queryid
+                       (or
```

Comment:
@awkay I am unclear if this is a workaround or a bugfix

Reply:
Good question. Dynamic queries were written before raw and hooks. I don't see how your change hurts anything. I mean, I guess we could allow a query id to be set when using the hook? That's kind of a double-level of dynamism I'm not sure is great, so let's go with this...